assets/src/edit-story/components/panels/link.js

```diff
@@ -53,10 +53,14 @@ const Note = styled.span`
 `;
 
 function LinkPanel({ selectedElements, onSetProperties }) {
+  const y = getCommonValue(selectedElements, 'y');
```

Comment:
Missed this before, but the `const link = getCommonValue(..., 'link')` - that would never yield "common value" since it's a freely-created object.